### PR TITLE
[master] build: prepare v0.16.0.0 release

### PR DESCRIPTION
**Pull request by selsta**
will close whichever PR we don’t need

### CODE REVIEW

**Comment by ph4r05** [comment]
@selsta do you pls have an idea when 0.16.0.0 is going to be released? Or when is the code freeze? I would need to add one simple PR regarding the Trezor integration once I know the Trezor firmware version supporting the newest changes.

Thanks!

**Comment by selsta** [comment]
@ph4r05 There is still 1 week left before we will tag.

**Comment by ph4r05** [comment]
And another thing - any idea when CLSAG stuff gets merged to the Monero? Could you pls notify me when that happens? Currently, I don't think there is an active PR with that. Thanks!

**Comment by selsta** [comment]
CLSAG has to get audited first, I would estimate ~ 3 months before releasing it. I will notify you once it gets PRed.

File: README.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Dates are provided in the format YYYY-MM-DD.
 | 1686275                        | 2018-10-19 | v9                | v0.13.0.0              | v0.13.0.4                  | bulletproofs required
 | 1788000                        | 2019-03-09 | v10               | v0.14.0.0              | v0.14.1.2                  | New PoW based on Cryptonight-R, new block weight algorithm, slightly more efficient RingCT format
 | 1788720                        | 2019-03-10 | v11               | v0.14.0.0              | v0.14.1.2                  | forbid old RingCT transaction format
-| 1978433                        | 2019-11-30* | v12                | v0.15.0.0              | v0.15.0.0                  | New PoW based on RandomX, only allow >= 2 outputs, change to the block median used to calculate penalty, v1 coinbases are forbidden, rct sigs in coinbase forbidden, 10 block lock time for incoming outputs
+| 1978433                        | 2019-11-30* | v12                | v0.15.0.0              | v0.16.0.0                  | New PoW based on RandomX, only allow >= 2 outputs, change to the block median used to calculate penalty, v1 coinbases are forbidden, rct sigs in coinbase forbidden, 10 block lock time for incoming outputs
```
Comment (MaxXor) [inline README.md:135]: Is the star (*) behind the date `2019-11-30*` still needed, as the fork has happened and is not an estimate anymore?

**Pull request merged**